### PR TITLE
Update research.rst

### PR DESCRIPTION
**Pull request by anranjiao**

### CODE REVIEW

File: docs/user/research.rst [hunk 1]
```diff
@@ -3,20 +3,31 @@ Research
...
...edu>`_, `Tongji University <https://en.tongji.edu.cn/>`_, `Xiamen University <https://en.xmu.edu.cn>`_, `China University of Geosciences <https://en.cug.edu.cn/>`_, `University of Kentucky <https://www.uky.edu/>`_, `Federal University of Rio de Janeiro <https://ufrj.br/en/>`_, `University of Johannesburg <https://www.uj.ac.za>`_, `University of Surrey <https://www.surrey.ac.uk>`_, `University of Los Andes <https://uniandes.edu.co/en>`_, `Clemson University <https://www.clemson.edu>`_, `Graz University of Technology <https://www.tugraz.at/en/home/>`_, `Nanchang University <https://english.ncu.edu.cn/>`_, `University of Nevada, Las Vegas <https://www.unlv.edu/>`_, `National University of Defence Technology <https://english.nudt.edu.cn/About/index.htm>`_, `University of Kaiserslautern <https://www.uni-kl.de/en/startseite>`_, `Shanghai Normal University <https://english.shnu.edu.cn>`_, `Worcester Polytechnic Institute <https://www.wpi.edu>`_, `Technical University of Cartagena <https://www.upct.es/english/content/departments/>`_, `University of Applied Sciences and Arts Northwestern Switzerland <https://www.fhnw.ch/en/startseiteu>`_, `Adolfo Ibáñez University <https://www.uai.cl/en/>`_
```
Comment (lululxvi) [inline docs/user/research.rst:6]: Could you count the number of universities? 

such as, change `universities` --> `> 30 universities`

The same for national labs.

File: docs/user/research.rst [hunk 2]
```diff
@@ -3,8 +3,8 @@ Research
...
```
Comment (lululxvi) [inline docs/user/research.rst:6]: `> 35 universities`

Comment (anranjiao) [reply docs/user/research.rst:6]: I think it's 36

Comment (lululxvi) [reply docs/user/research.rst:6]: use "> 35"

**Pull request merged**